commons/src/main/kotlin/com/simplemobiletools/commons/compose/theme/Theme.kt

```diff
@@ -0,0 +1,79 @@
+package com.simplemobiletools.commons.compose.theme
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.darkColorScheme
+import androidx.compose.material3.dynamicDarkColorScheme
+import androidx.compose.material3.dynamicLightColorScheme
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.SideEffect
+import androidx.compose.runtime.remember
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.LocalView
+import com.google.accompanist.systemuicontroller.rememberSystemUiController
+import com.simplemobiletools.commons.compose.extensions.config
+import com.simplemobiletools.commons.compose.theme.model.Theme
+import com.simplemobiletools.commons.compose.theme.model.Theme.Companion.systemDefaultMaterialYou
+import com.simplemobiletools.commons.helpers.isSPlus
+
+@Composable
+internal fun Theme(
+    theme: Theme = systemDefaultMaterialYou(),
+    content: @Composable () -> Unit,
+) {
+    val view = LocalView.current
+    val context = LocalContext.current
+    val systemUiController = rememberSystemUiController()
+    val baseConfig = remember { context.config }
+
+    val colorScheme = if (!view.isInEditMode) {
+
```

Comment:
avoid such empty lines please. Also having "val colorScheme" twice under each other is really confusing